src/colvar_neuralnetworkcompute.h

```diff
@@ -20,7 +20,7 @@
 #ifdef LEPTON
 #include "Lepton.h"
 #endif
-
+enum LayerType {DENSE, SELFATTENTION, INVALID};
```

Comment:
Will it be better to move the enum definition inside namespace neuralnetwork CV?